visualization/tools/librealsense_viewer.cpp

```diff
@@ -98,16 +115,6 @@ class LibRealSenseViewer
     LibRealSenseViewer (pcl::LibRealSenseGrabber& grabber)
     : grabber_ (grabber)
     , viewer_ ("LibRealSense Viewer")
-    , grabber_second_ (grabber)
```

Comment:
That makes sense. Let's focus this sample only for one grabber case. BTW, does your grabber implementation supports multiple grabbers for one camera device?